.github/workflows/upload-ci-logs-v2.yml

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# see https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#create-a-commit-status
+name: Nvfuser-CI Logs v2
+on:
+  workflow_dispatch:
+    inputs:
+      state:
+        description: 'job status'
+        required: true
+      descr:
+        description: 'description of the job'
+        required: true
+      commit_sha:
+        description: 'SHA of the commit that was tested.'
+        required: true
+      target_url:
+        description: 'target url'
+        required: true
+      context:
+        description: 'context'
+        required: true
+      pr_number:
+        description: 'pr number'
+        required: false
```

Comment:
nitpick, description isn't very helpful here since they are mostly just the field name. i.e. what is a `context`?

Reply:
context is the message with small font size italic grey text after each GitHub action name, e.g. "successful in 3m"